pkg/controller/studyjob/const.go

```diff
@@ -0,0 +1,23 @@
+/*
+ Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package studyjob
+
+const (
+	DefaultJobWorker = "Job"
+	TFJobWorker = "TFJob"
+)
+
+const (
+	WorkerState_Active = 0
```

Comment:
This is a generic way to represent the state of a worker, which can be either a batch job or TF job. In lines 415-454 we compute the current worker status and pass the handling to updateWorker(). This is to avoid duplicating the long code section for each job type.

Reply:
Sure, fixed.